README.md

```diff
@@ -16,3 +16,11 @@ The projects for these repositories are:
 
 Please visit the appropriate project in the above list for obtaining additional information,
 asking questions, or opening issues.
+
+## Version Tagging Scheme
+
+Updates to the `Vulkan-Headers` repository which correspond to a new Vulkan specification release are tagged using the following format: `v<`_`version`_`>` (e.g., `v1.1.96`).
+
+**Note**: Marked version releases have undergone thorough testing but do not imply the same quality level as SDK tags. SDK tags follow the `sdk-<`_`version`_`>.<`_`patch`_`>` format (e.g., `SDK-1.1.92.0`).
```

Comment:
Should probably use lower-case for "SDK" after the "e.g.,"

Reply:
Good catch! This is now updated.